docs/pages/server-access/guides/openssh.mdx

```diff
@@ -117,11 +146,44 @@ host.
 
 ### Issue a host certificate
 
-On your local machine, assign the IP address or fully qualified domain name of
-your Node to an environment variable.
+When creating host certificates, it is important to specify all the domain names
```

Comment:
- We're no longer capitalizing "Node" in the docs to refer to an SSH Service instance
- I'm not sure if this paragraph belongs in this location. We haven't shown the user how to create a host certificate yet, so they might get the impression that we're leaving them to figure this out for themselves. Can we put these instructions by the `tctl auth sign` instructions?

Reply:
Yeah that's a better place for it, done